versioned_docs/version-v2.18.x/user-guide/address-storage-requirements.md

```diff
@@ -5,15 +5,15 @@
 
```

Comment:
@1000TurquoisePogs - This file has different content in v2.18 and v3.1. Please confirm that  is supposed to be the case. If not, please advise what needs to be changed and in which version.